source/chapters/user_interface.rst

```diff
@@ -871,8 +871,20 @@ Equalizers and Gain Knobs
    EQ Controls
 
 **Gain Knob**
+  Gain or Loss in sounds is also referred to as volume levels.
   The gain knob allows you to adjust the gain applied to the deck.
   Use this to compensate for the differences in recording levels between tracks.
+  There are 4 Gain Knobs:
+      1. The Super-Knob
+      2. The Low Filter Knob
+      3. The Mid Filter Knob
+      4. The High Filter Knob
+  Each knob has a left and right side. The left side directs sounds to the left speakers or headset. The right side directs sounds to the right speakers or headset.
+   The mid-position places the filter in a balanced position. This allows the filter to be shared equally by both the left and right sides of the headset.
```

Comment:
wrong. EQ knobs affect the volume of certain frequency ranges. Balance is something totally different and balance controls are located only in the Master section of the mixer.
please use Mixxx yourself to test what the controls do and to pick additonal info from the tooltips (see above comment)